lib/constable_web/views/error_view.ex

```diff
@@ -1,16 +1,16 @@
 defmodule ConstableWeb.ErrorView do
-  use Constable.Web, :view
+  use ConstableWeb, :view
 
-  def render("404.html", _assigns) do
-    "Page not found - 404"
-  end
-
-  def render("500.html", _assigns) do
-    "Server internal error - 500"
-  end
+  # If you want to customize a particular status code
+  # for a certain format, you may uncomment below.
+  # def render("500.html", _assigns) do
+  #   "Internal Server Error"
+  # end
 
-  # Render all other templates as 500
-  def render(_, assigns) do
```

Comment:
I think that a side of this change will be to handle this better - https://app.honeybadger.io/projects/48229/faults/39407168